draft-ietf-tls-rfc4492bis.xml

```diff
@@ -339,7 +340,10 @@
             deprecates all but the uncompressed point format. Implementations of this document MUST support the
             uncompressed format for all of their supported curves, and MUST NOT support other formats for curves
             defined in this specification. For backwards compatibility purposes, the point format list extension
-            MUST still be included, and contain exactly one value: the uncompressed point format (0).</t>
+            MAY still be included, and contain exactly one value: the uncompressed point format (0). RFC 4492 
```

Comment:
For a while there we had an interception gateway that didn't send the point format extension. We didn't get any complaints about that. I don't have the statistics from that anymore, so I don't know if servers that otherwise would choose ECDHE ciphersuites chose RSA ciphersuites instead, but it did work with at least common servers (Google, Facebook, several banks)